src/lib.rs

```diff
@@ -1040,7 +1046,7 @@ mod tests {
         }
 
         // Combined, they produce a signature matching the main public key.
-        let sig = pk_set.combine_signatures(&sigs).expect("signatures match");
+        let sig = pk_set.combine_signatures(sigs).expect("signatures match");
```

Comment:
Is this a breaking change?

Reply:
no, because for BTreeMap, one can also pass `&sigs` as before, or also it works to pass `sigs`.  For Vec, only the latter.  See for example line 1060 which hasn't changed:

```
pk_set.combine_signatures(&sigs2).expect("signatures match");
```

Reply:
I just verified that the previous usage/workaround for Vec still works also:

```
        let sigs_ref: Vec<(usize, &SignatureShare)> = sigs.iter().map(|(i, sig)| (*i, sig)).collect();
        let sig = pk_set.combine_signatures(sigs_ref).expect("signatures match");
```